indolaw-nextjs/utils/grammar.tsx

```diff
@@ -98,7 +99,8 @@ export interface Complex {
 export function renderStructure(
   structure: Complex | Primitive,
   key?: string | number,
-  isMobile?: boolean,
+  penjelasanUmum?: Array<Complex | Primitive>,
+  isMobile?: boolean
```

Comment:
fine for now, but if the arguments gets any more complicated, maybe we should have it take it an object as an argument instead? I can imagine we might have super ugly callsites in the future like this hahaha:

```
renderStructure(
    structure, 
    key, 
    [], // wtf is this argument supposed to be
    false // wtf is this boolean supposed to mean
)
```

so at some point we may want to change it to 
```
renderStructure({
    structure: structure, 
    key: key, 
    penjelasanUmum: [], // wtf is this argument supposed to be
    isMobile: false // wtf is this boolean supposed to mean
})
```

open to thoughts/suggestions here